src/Modules/OrchardCore.Commerce/Services/StripePaymentService.cs

```diff
@@ -277,16 +253,38 @@ public async Task UpdateOrderToPaymentFailedAsync(PaymentIntent paymentIntent)
         return order;
     }
 
+    public long GetPaymentAmount(decimal defaultTotalValue, string currencyType)
+    {
+        long amountForPayment;
+
+        // If I convert it to conditional expression, it will warn me to extract it again.
+#pragma warning disable IDE0045 // Convert to conditional expression
+        // We need to convert the value (decimal) to long.
+        // https://stripe.com/docs/currencies#zero-decimal
+        if (CurrencyCollectionConstants.ZeroDecimalCurrencies.Contains(currencyType))
+        {
+            amountForPayment = (long)Math.Round(defaultTotalValue);
```

Comment:
Instead of defining a temporary variable you could return the value directly. (I think if you do that you won't have that IDE0045 warning either.)

Reply:
This was code that I just moved into a method, but I'll update it accordingly.

Reply:
Thanks!